helmfile.d/0400.kube-prometheus.yaml

```diff
@@ -159,6 +159,71 @@ releases:
 {{ end }}
 
 
+#######################################################################################
+## kube-prometheus-pushgateway                                                       ##
+## Provide kube-prometheus default push gateway                                      ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/coreos/prometheus-operator/tree/master/helm/kube-prometheus
+#   - https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus
+#   - https://github.com/helm/charts/tree/master/stable/prometheus-pushgateway
+#
+- name: "kube-prometheus-pushgateway"
+  namespace: "monitoring"
+  labels:
+    chart: "prometheus-pushgateway"
+    component: "monitoring"
+    namespace: "monitoring"
+    vendor: "kubernetes"
+    default: 'true'
+  chart: "stable/prometheus-pushgateway"
+  version: "0.1.2"
+  wait: true
+  values:
+  - serviceAccount:
+      create: {{ env "KUBE_PROMETHEUS_SERVICE_ACCOUNT_CREATE" | default "false" }}
+    replicaCount: {{ env "KUBE_PROMETHEUS_PUSH_GATEWAY_REPLICA_COUNT" | default "1" }}
+
+#######################################################################################
+## kube-prometheus-pushgateway-monitor                                               ##
+## Provide kube-prometheus service monitor for  push gateway                         ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/coreos/prometheus-operator/tree/master/helm/kube-prometheus
+#   - https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus
+#
+- name: "kube-prometheus-pushgateway-monitor"
+  namespace: "monitoring"
+  labels:
+    chart: "prometheus-pushgateway"
```

Comment:
why it's the same `chart: "prometheus-pushgateway"` as above?

Reply:
`monochart` is too common. 
But mean this release is addition to `prometheus-pushgateway` that intergrates it with `kube-prometheus` and should be installed togeather